Sources/SwiftFormat/PrettyPrint/PrettyPrintBuffer.swift

```diff
@@ -118,7 +118,13 @@ struct PrettyPrintBuffer {
     writeRaw(text)
     consecutiveNewlineCount = 0
     pendingSpaces = 0
-    column += text.count
+
+    // In case of comments, we may get a multi-line string.
+    // To account for that case, we need to correct the lineNumber count.
+    // The new column is only the position within the last line.
+    let lines = text.split(separator: "\n")
+    lineNumber += lines.count - 1
+    column += lines.last?.count ?? 0
```

Comment:
If there is a newline in the text, should we set `column` to `lines.last.count` instead of adding it? A newline effectively resets the column to 0 (see eg. `writeNewlines`).

Reply:
Good point.
If there is 
* 0 new lines, we need to add
* 1 or more new lines, we need to set it to lines.last.count.

Reply:
I pushed an update.